src/components/Navigation.tsx

```diff
@@ -146,19 +146,23 @@ export const Navigation = (): JSX.Element => {
         >
           <h2 style={{ margin: '0 0.55rem 0 0.8rem' }}>Village-monde </h2>
           {village &&
-            village.countries.map((country: Country) => (
-              <Flag
-                style={{ margin: '0.25rem' }}
-                key={country.isoCode}
-                country={country.isoCode}
-                isMistery={
-                  !village ||
-                  !user ||
-                  (selectedPhase === 1 && user.country.isoCode.toUpperCase() !== country.isoCode && !isModerateur) ||
-                  (user.firstLogin < 2 && user.country.isoCode.toUpperCase() !== country.isoCode && !isModerateur)
-                }
-              ></Flag>
-            ))}
+            village.countries.map((country: Country) =>
+              user?.type === UserType.OBSERVATOR && selectedPhase === 2 ? (
+                <Flag style={{ margin: '0.25rem' }} key={country.isoCode} country={country.isoCode} />
+              ) : (
+                <Flag
+                  style={{ margin: '0.25rem' }}
+                  key={country.isoCode}
+                  country={country.isoCode}
+                  isMistery={
+                    !village ||
+                    !user ||
+                    (selectedPhase === 1 && user.country.isoCode.toUpperCase() !== country.isoCode && !isModerateur) ||
+                    (user.firstLogin < 2 && user.country.isoCode.toUpperCase() !== country.isoCode && !isModerateur)
+                  }
+                ></Flag>
+              ),
+            )}
```

Comment:
Instead of adding a new logic with the ternary condition, shouldn't we only change the `isModerateur` condition to `isObservator` ? 

```typescript
<Flag
  style={{ margin: '0.25rem' }}
  key={country.isoCode}
  country={country.isoCode}
  isMistery={
    !village ||
    !user ||
    (selectedPhase === 1 && user.country.isoCode.toUpperCase() !== country.isoCode && !isObservator) ||
    (user.firstLogin < 2 && user.country.isoCode.toUpperCase() !== country.isoCode && !isObservator)
  }
></Flag>
```